src/MeetingsSDKAdapter.js

```diff
@@ -376,16 +396,25 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     const sdkMeeting = this.fetchMeeting(ID);
 
     try {
+      const isInSession = this.meetings[ID].remoteVideo !== null;
```

Comment:
It might be worthwhile to make this a helper function like `isMeetingJoined(ID)`

Reply:
this should be implemented once we have a legit property - AKA state/meetingState